src/components/autocomplete.ts

```diff
@@ -227,14 +226,14 @@ const actions = {
     // TODO: what are we doing with detail and documentation?
     // show both? or one or the other?
 
-    if (!detail || !documentation)
-      (async () => {
-        // TODO: what do with .detail?
-        const details = await api.ai.completions.getDetail(completionItem)
-        if (!details || !details.documentation) return
-        const richFormatDocs = await parseDocs(details.documentation)
-        a.showDocs(richFormatDocs)
-      })()
+    // if (!detail || !documentation)
```

Comment:
remove comment (if it's unnecessary to keep the code; I just didn't know/don't know what it was doing)

Reply:
I think getting documentation may be slow (you don't want to get it for _all_ of the possibly many returned results at the same time), so async does make sense.